tests/end2end-tests/custom-gnn/train.py

```diff
@@ -108,7 +108,7 @@ def main(args):
                            help="The name of the label.")
     argparser.add_argument("--num-classes", type=int, required=True,
                            help="The number of classes.")
-    argparser.add_argument("--local_rank", type=int,
+    argparser.add_argument("--local-rank", "--local_rank", type=int,
```

Comment:
Typo?

Reply:
Nope. local-rank is for torch >= 2.0.0, local_rank is for lower version torch.

Reply:
I c.

Reply:
Can you add a comment for that?